tools/demo.py

```diff
@@ -178,7 +178,7 @@ def visual(self, output, img_info, cls_conf=0.35):
         bboxes /= ratio
 
         cls = output[:, 6]
-        scores = output[:, 4] * output[:, 5]
+        scores = torch.sqrt(output[:, 4] * output[:, 5])
```

Comment:
This line shall be updated in a separate PR, since it serves different purpose with the other modifications.